scout/adapter/mongo/case.py

```diff
@@ -246,6 +246,22 @@ def cases(self, owner=None, collaborator=None, query=None, skip_assigned=False,
         else:
             return self.case_collection.find(query).sort('updated_at', -1)
 
+    def prioritized_cases(self, institute_id=None):
```

Comment:
This calls for a nice test, but you already knew it! ;)

Reply:
😆 I will not reach 100% tonight, but here is a contribution at least!